src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -64,6 +64,11 @@ internal static class CompletionOptions
             new(nameof(CompletionOptions), nameof(TriggerInArgumentLists), defaultValue: null,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.TriggerInArgumentLists"));
 
+        // Use tri-value so the default state can be used to turn on the feature with experimentation service.
+        public static readonly PerLanguageOption2<bool?> AutomaticallyAddParenthesisBySemicolon =
```

Comment:
My understanding is the option was going to be added for <kbd>Tab</kbd>, but no option was going to be added for <kbd>;</kbd> or <kbd>.</kbd>.

Reply:
Removed